builder/processor/src/main/java/io/helidon/builder/processor/Types.java

```diff
@@ -33,41 +33,72 @@ final class Types {
     static final String RUNTIME_OBJECT = "io.helidon.builder.api.RuntimeType.Api";
     static final String RUNTIME_PROTOTYPE = "io.helidon.builder.api.RuntimeType.PrototypedBy";
     static final String PROTOTYPE = "io.helidon.builder.api.Prototype.Api";
-    static final String PROTOTYPE_SAME_GENERIC = "io.helidon.builder.api.Prototype.SameGeneric";
     static final String BUILDER_DECORATOR = "io.helidon.builder.api.Prototype.BuilderDecorator";
     static final String IMPLEMENT = "io.helidon.builder.api.Prototype.Implement";
-    static final String CONFIDENTIAL = "io.helidon.builder.api.Prototype.Confidential";
-    static final String REDUNDANT = "io.helidon.builder.api.Prototype.Redundant";
     static final String FACTORY_METHOD = "io.helidon.builder.api.Prototype.FactoryMethod";
     static final String PROTOTYPE_BUILDER = "io.helidon.builder.api.Prototype.Builder";
+    static final String PROTOTYPE_CONFIGURED = "io.helidon.builder.api.Prototype.Configured";
     static final String PROTOTYPE_CONFIGURED_BUILDER = "io.helidon.builder.api.Prototype.ConfiguredBuilder";
     static final String PROTOTYPE_BLUEPRINT = "io.helidon.builder.api.Prototype.Blueprint";
     static final String PROTOTYPE_FACTORY = "io.helidon.builder.api.Prototype.Factory";
     static final String PROTOTYPE_ANNOTATED = "io.helidon.builder.api.Prototype.Annotated";
     static final String CUSTOM_METHODS = "io.helidon.builder.api.Prototype.CustomMethods";
     static final String PROTOTYPE_CUSTOM_METHOD = "io.helidon.builder.api.Prototype.PrototypeMethod";
     static final String BUILDER_CUSTOM_METHOD = "io.helidon.builder.api.Prototype.BuilderMethod";
-    static final String SINGULAR = "io.helidon.builder.api.Prototype.Singular";
+    static final String DESCRIPTION = "io.helidon.builder.api.Description";
```

Comment:
i wish there was a better way to declare these in java. this is fragile.

Reply:
yes, but dependency on the module is worse. We want to keep the number of dependencies of annotation processors as low as possible